e2e/tests/components/logical-expression-builder/logical-expression-builder.po.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { $, $$, browser, ElementArrayFinder, ElementFinder } from 'protractor';
+import { Expression } from '../../../../src/components/logical-expression-builder';
```

Comment:
This should be imported from @ux-aspects/ux-aspects. I'm surprised this was not a build error.